src/integration-tests/test_reconfigure_domains.py

```diff
@@ -49,24 +49,28 @@ def setup_cluster(self, cluster: Cluster):
         self.reader.open(URI_PRIORITY_1, flags=["read"], succeed=True)
         self.reader.open(URI_PRIORITY_2, flags=["read"], succeed=True)
 
-    # Instruct 'writer to 'POST 'n' messages to the domain.
-    # Returns 'True' if all of them succeed, and a false-y value otherwise.
+    # Instruct writer to POST `n` messages to the `uri`.  Return true if all of
+    # them succeed, else a false-y value.
     def post_n_msgs(self, uri, n):
         results = (
             self.writer.post(uri, payload=[f"msg{i}"], wait_ack=True)
             for i in range(0, n)
         )
         return all(res == Client.e_SUCCESS for res in results)
 
-    # Helper method which tells 'leader' to reload the domain config.
-    def reconfigure_to_n_msgs(self, cluster: Cluster, num_msgs, leader_only=True):
+    # Instruct the leader to reconfigure domain config in `cluster` to have a
+    # limit of `max_num_msgs`.  If 'leader_only' is true, the 'DOMAINS
+    # RECONFIGURE' command will only be issued to the leader.  Return true if
+    # reconfigure succeeds, else false.
+    def reconfigure_to_limit_n_msgs(self, cluster: Cluster, max_num_msgs):
```

Comment:
```suggestion
    def reconfigure_to_limit_n_msgs(self, cluster: Cluster, max_num_msgs: int) -> bool:
```
Might add more type hints